<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormDoctor</class>
 <widget class="QDialog" name="FormDoctor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Введите номер телефона </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Введите специальность</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_full_name">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_email">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Введите почтовый адрес </string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Введите ФИО</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_speciality">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_phone">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
